editor/i18n/en/localization.js

```diff
@@ -377,6 +389,17 @@ module.exports = {
         handleTouchEvent:
             'Once checked, the RichText will block all input events (mouse and touch) within the bounding box of the node, preventing the input from penetrating into the underlying node.',
     },
+    UICoordinateTracker: {
+        target: 'Target node',
+        camera: '3D camera for conversion',
+        use_scale: 'Do you need to scale the converted 2d node size according to how far the 3D node is from the camera',
+        distance: 'The distance from the camera is the normal display calculation size',
+        sync_events: 'Event callback after value change\nThe first parameter of the callback is the mapped local coordinate\nThe second is the distance from the camera',   
```

Comment:
Event callback after coordinates synchronization.
The first parameter of the callback is the mapped local coordinate in UI camera.
The second parameter is the distance scale of the 3d node from the 3d camera viewport.